config.json

```diff
@@ -115,6 +115,17 @@
       ]
     },
     {
+      "uuid": "40d4bb98-2e4a-4579-810e-g1230bc158e9",
+      "slug": "complex-numbers",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+        "Mathematics",
```

Comment:
I think I will feel the urge to remove this. I will feel bad having this exercise with "mathematics" topic, and none of the other exercises that should have "mathematics" topic have it. It isn't one that would affect a placement decision (the topics added in https://github.com/exercism/haskell/pull/476 ). 

I guess I would accept a PR that adds the mathematics topic to all the exercises that should have it.

Reply:
Maybe I'll do that.